packages/core/src/contexts/reducers/store.ts

```diff
@@ -60,6 +61,9 @@ enum PreferencesDispatchAction {
   UPDATE_ALTERNATE_CONTACT_NAMES = 'preferences/updateAlternateContactNames',
   AUTO_LOCK_TIME = 'preferences/autoLockTime',
   SET_THEME = 'preferences/setTheme',
+  SET_SELECTED_MEDIATORS = 'preferences/setSelectedMediators',
```

Comment:
```suggestion
  SET_SELECTED_MEDIATOR = 'preferences/setSelectedMediator',
```
Nit, but we can only select one at a time, right?